### PR TITLE
Created Signup Page 

### PR DESCRIPTION
**Pull request by rahulcs754**
* Added signup page
image

Please check and give some suggestion

### CODE REVIEW

File: assets/css/custom.css [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    width: 70%;
+}
+
+.width-60{
+    width: 60%;
+}
+
+.width-50{
+    width: 50%;
+}
```
Comment (kjais1720) [inline assets/css/custom.css]: both, the `.window-`  & `.width-` classes perform the same task, you can remove one of those

File: assets/css/custom.css [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+.window-50{
+    width: 50%;
+}
+
+
+.width-100{
+    width: 70%;
+}
```
Comment (kjais1720) [inline assets/css/custom.css]: is it a typo?

File: assets/css/custom.css [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+.text-filter{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 1rem;
+}
+
+.mycart-amount-list{
+    display: flex;
+    flex-direction:column ;
```
Comment (kjais1720) [inline assets/css/custom.css]: you can use the utility class for `display: flex; flex-direction: column;`

File: assets/css/custom.css [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+    gap: 2px;
+}
+.mycart-amout-desc{
+    display: flex;
+    flex-direction: row;
```
Comment (kjais1720) [inline assets/css/custom.css]: `flex-direction: row;` is the default values for flexbox, you don't need to specify it here

File: assets/css/custom.css [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+    justify-content: space-between;
+}
+
+.section {
+    display: flex;
+    justify-content: center;
+    align-content: center;
```
Comment (kjais1720) [inline assets/css/custom.css]: use utility class for this

File: assets/css/custom.css [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+}
+
+/* Login css */
+
+.login-box,
+.signup-box{
+    display: flex;
+    flex-direction:row ;
+    justify-content: center;
+    align-items: center;
```
Comment (kjais1720) [inline assets/css/custom.css]: The `flex-direction: row` is not needed and you can use utility class for the other properties

**Review by kjais1720 — CHANGES_REQUESTED** [review]

File: assets/css/custom.css (hunk 1 above)
Comment (rahulcs754) [reply assets/css/custom.css]: Hi removed these both and add these classes.

File: assets/css/custom.css (hunk 2 above)
Comment (rahulcs754) [reply assets/css/custom.css]: Hi removed these both and add these classes. to  utilities

File: assets/css/custom.css (hunk 3 above)
Comment (rahulcs754) [reply assets/css/custom.css]: yes i changed my code that you suggest to me

File: assets/css/custom.css (hunk 4 above)
Comment (rahulcs754) [reply assets/css/custom.css]: okay

File: assets/css/custom.css (hunk 5 above)
Comment (rahulcs754) [reply assets/css/custom.css]: okay

File: assets/css/custom.css (hunk 1 above)
Comment (rahulcs754) [reply assets/css/custom.css]: remove done and i used this into utilities class

File: assets/css/custom.css (hunk 4 above)
Comment (rahulcs754) [reply assets/css/custom.css]: this change in cart page i will do that

File: assets/css/custom.css (hunk 6 above)
Comment (rahulcs754) [reply assets/css/custom.css]: removed

**Review by kjais1720 — APPROVED** [review]
All good then👍

File: pages/signup.html [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+                    </div>
+                    
+                </form>
+        
+
+            </div>
+            
+          
+   
+
```
Comment (coldpigli) [inline pages/signup.html]: A lot of redundant spaces

**Review by coldpigli — APPROVED** [review]
Everything looks okay except for some redundant spaces in the html

File: pages/signup.html (hunk 7 above)
Comment (rahulcs754) [reply pages/signup.html]: remove spaced as you said

**Pull request merged**